### ISSUE TITLE
Object rest spread fails to compile when imported from node module

### ISSUE THREAD

**Issue by aaemnnosttv** [JeffreyWay/laravel-mix#1805]
- Laravel Mix Version: 2.1.14
- Node Version (`node -v`):  v10.13.0
- NPM Version (`npm -v`): 6.4.1
- OS: Mojave

### Description:

Given an import in a file compiled with `mix.js()` like so

```
import 'bootstrap/js/src/collapse';
```

produces an error indicating an inability to compile the rest spread operator

```
Module parse failed: Unexpected token (271:8)
You may need an appropriate loader to handle this file type.
|     _getConfig(config) {
|       config = {
|         ...Default,
|         ...config
|       }
 @ ./src/index.js 1:0-35
 @ multi ./src/index.js
```

As noted on the issue repo below, this error does _not_ happen if the source code is all "local" (not imported from `node_modules`).

### Steps To Reproduce:

https://github.com/aaemnnosttv/laravel-mix-rest-spread-issue

---

What am I doing wrong?

Related #76, #1674

**Comment by narrative1st** [comment]
same issue here....

**Comment by JeffreyWay** [comment]
Yes, that's exactly the issue. Short of overriding the generated webpack config object, this can't be changed at the moment.

**Comment by cameronscott137** [comment]
Hey y'all, if you're like me and came to this issue a few years after it was closed, you can now update to Laravel Mix ^5.0, and that will resolve it. No extra plugins or .babelrc files required.

**Comment by gavinhewitt** [comment]
> Hey y'all, if you're like me and came to this issue a few years after it was closed, you can now update to Laravel Mix ^5.0, and that will resolve it. No extra plugins or .babelrc files required.

Sadly that is not what I'm experiencing. Without .babelrc I still get the unexpected token error. With .babelrc it compiles without problems. However, this creates a new problem, which is that the built files are somehow not working. They load but there's output. I'm creating a new custom field for Laravel Nova that imports tiptap editor. If I build without the imports, the field is shown. If I build with the imports, nothing :-(
